lib/resources/api_resource.rb

```diff
@@ -22,16 +22,8 @@ def self.authorize_bearer_header(frame_token)
       end
     end
   
-    def self.authorize_basic_auth_header
-      { Authorization: "Basic #{self.encoded_secret}" }
-    end
-  
-    def self.encoded_secret
-      Base64.strict_encode64("#{ENV['FRAMEIO_CLIENT_ID']}:#{ENV['FRAMEIO_CLIENT_SECRET']}")
-    end
-
-    def self.request(method, url, headers, body = {})
-      HTTParty.send(method, url, :headers => headers, :body => body)
+    def self.request(method, url, frameio_token:, body = {})
```

Comment:
The frameio_token argument should not be a keyword arg.

```suggestion
    def self.request(method, url, frameio_token, body = {})
```